run.py

```diff
@@ -17,6 +17,7 @@
 
 DEFAULT_BRANCH = '8.0'
 
+
```

Comment:
IIRC on xcp-ng-tests we disabled that rule, but I won't complain for a single added line.